environment.go

```diff
@@ -405,8 +405,10 @@ func (e *envState) initSenders() error {
 
 	if slack := &e.settings.Slack; len(slack.Token) != 0 {
 		sender, err = send.NewSlackLogger(&send.SlackOptions{
-			Channel: "#",
-			Name:    "evergreen",
+			Channel:  "#",
```

Comment:
I don't understand the `#`, and it's not obvious to me why this works. Maybe a comment?